### PR TITLE
Set DBF default character encoding to ISO-8859-1

### PR DESCRIPTION
**Pull request by icetan**
Got some encoding problems when adding å/ä/ö in my shape data. Turns out the only supported encoding for DBF (by the original spec.) is ISO-8859-1.

There have been some extensions to the specifications and I tried to accommodate them as well.

I had to add a dependency on Iconv to decode ISO-8859-1.

This was a helpful post: http://gis.stackexchange.com/questions/3529/which-character-encoding-is-used-by-the-dbf-file-in-shapefiles

### CODE REVIEW

**Comment by icetan** [comment]
I think I accidentally submitted multipolygon support in this request as well.

**Comment by yuletide** [comment]
Awesome.

File: package.json [hunk 1]
```diff
@@ -1,17 +1,20 @@
 {
   "name" : "shp",
   "description" : "Convert shapefiles into geoJson without ogr2ogr/GDAL",
-  "version" : "0.0.1",
+  "version" : "0.0.2",
   "repository" : {
     "type" : "git",
-    "url" : "git://github.com/yuletide/node-shp"
```
Comment (yuletide) [inline package.json:7]: url should still be set to this repo, also, if you could separate this into a pull req for each feature that'd be great. thanks for contributing!

**Comment by icetan** [comment]
Yupp, sorry about that. I'm closing this pull and making two new.

**Pull request closed without merging**